### ISSUE TITLE
Issue with Embedding Blujeans SDK

### ISSUE THREAD

**Issue by richieferns** [bluejeans/webrtc-embed-sdk#5]
Recently we have been facing issue with embedding Bluejeans in a page. Issue details in screenshot below.
![image](https://user-images.githubusercontent.com/20200767/106005204-fa649e80-60d9-11eb-9861-f2f6ec965eed.png)

**Comment by grihan6** [comment]
@naveenkbhaskar Can you please check on this issue?

**Comment by abrindam** [comment]
This is likely caused by having the setting "Block third-party cookies and
site data" enabled in Chrome.

https://www.chromium.org/for-testers/bug-reporting-guidelines/uncaught-securityerror-failed-to-read-the-localstorage-property-from-window-access-is-denied-for-this-document

Can you please check this setting? If it is important that our Embed SDK
work even if this setting is enabled, let us know.

**Comment by NSMQA** [comment]
Yes the setting is enabled in chrome. "Block third-party cookies in Incognito". Bluejeans (embedded) works when allow all cookies is selected.

**Comment by renegadephantom** [comment]
We are actively working on a fix. very soon we will have  issue resolved.
